internal/witness/cmd/witness/config/config.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package main is for populating the witness config file according to a set
```

Comment:
```suggestion
// config is a tool to populate the witness config file according to a set
```

Reply:
There's a different rule for main files (or at least, a more permissive one that _also_ allows for this more sensical description).

Reply:
Done!